plugin/kubernetes/README.md

```diff
@@ -65,6 +66,16 @@ kubernetes [ZONES...] {
      option requires substantially more memory than in insecure mode, since it will maintain a watch
      on all pods.
 
+* `endpoints` **ENDPOINT-MODE** sets the mode for handling service endpoint A
+   records, e.g.
+   `pod-name.my-service.svc.cluster.local. in A 1.2.3.4` if the pod is an
+   endpoint of the service `my-service`.  By default, these will not resolve.
+   Valid values for **ENDPOINT-MODE**:
+
+   * `name`: Return an A record if there exists a pod that is a valid endpoint
+    for the service.  This only works if `pods` is set to either `insecure` or
+    `verified`.  If `hostname` is set on a pod, then it always has precedence.
```

Comment:
This should work regardless of the pods setting, I believe. We always watch endpoints, and we respond to their hostname records regardless of whether `pod` types are enabled.

We could add an option to `pods verified` that enables `pod-name.namespace.pod.cluster.local`.

Reply:
But that would be a separate PR...

Reply:
@johnbelamaric you are correct, it works if `pods` is disabled. Updating doc.